frontend/src/general/NotFound.js

```diff
@@ -1,12 +1,38 @@
 import React from "react";
+import { useHistory } from "react-router";
+import Button from "@material-ui/core/Button";
 import messages from "./textHolder";
+import style from "./NotFound.module.css";
+import changePath from "./helperFunctions";
+import { ReactComponent as Logo2 } from "./Logo2.svg";
+import { ReactComponent as NotFound2 } from "./NotFoundImage.svg";
 
 const notFoundMessage = messages.notFound;
 
-const NotFound = () => (
-  <div>
-    {notFoundMessage.message}
-  </div>
-);
+const NotFound = () => {
+  const history = useHistory();
+  return (
+    <div className={style.notFoundPageContainer}>
+      <div className={style.header}>
+        <div className={style.titleContainer}>
+          <div className={style.logo}>
+            <Logo2 />
+          </div>
+        </div>
+      </div>
```

Comment:
Is it necessary to have 3 divs here?

Reply:
I added them to be consistent with the style in the LandingPage, but I can try improve it after the demo!